packages/components/src/DateTimePickers/views/common/ViewLayout/ViewLayout.scss

```diff
@@ -0,0 +1,28 @@
+
+.container {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+
+    .header {
+        display: flex;
+        height: 5rem;
+
+        .left, .right {
+            flex: 1;
+        }
+
+        .middle {
+            flex: 3;
+        }
+    }
+
+    .body {
+        flex: 1;
+    }
+
+    .node-container > * {
```

Comment:
`node-container` is not really explicit. Perhaps something like `header-parts` ?

Reply:
It's common also with the body. I named it _node_ because each spot are designed to receive a react thing, and actually it sounds like it's an _element_ not a _node_...
And each spot are container of the element. So **element-container** ? Still a general carryall name but in the other hand it's design to be generic...

Reply:
&#10003;